lib/packwerk/railtie.rb

```diff
@@ -0,0 +1,13 @@
+require 'packwerk'
+require 'rails'
+
+module Packwerk
+  class Railtie < Rails::Railtie
+    railtie_name :packwerk
+
+    rake_tasks do
+      path = File.expand_path(__dir__)
+      Dir.glob("#{path}/tasks/**/*.rake").each { |f| load f }
```

Comment:
We could probably be more explicit and just add the specific rake task we want to include.

I used this as a reference: https://gist.github.com/ntamvl/7a6658b4cd82d6fbd15434f0a9953411